### PR TITLE
User sign in/out

### PR DESCRIPTION
**Pull request by DmitriyFirsov**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+POSTGRES_PASSWORD=some-password
+POSTGRES_USER=test-app-user
+POSTGRES_DB=todo-app
+POSTGRES_HOST=database
```
Comment (KilinMikhail) [inline .env.example]: it is good practice to add empty line in the end of files

File: .rubocop.yml [hunk 2]
```diff
@@ -10,6 +10,10 @@ AllCops:
     - vendor/**/*
     - .git/**/*
 
+Metrics/MethodLength:
```
Comment (KilinMikhail) [inline .rubocop.yml]: why do you need to exlude this path?

File: app/views/devise/shared/_links.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
```
Comment (KilinMikhail) [inline app/views/devise/shared/_links.html.erb:5]: ok, but in the future it should be in policy (one of the future tasks)

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -7,7 +7,6 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (KilinMikhail) [inline app/views/layouts/application.html.erb:10]: why you removed this line?

Comment (DmitriyFirsov) [reply app/views/layouts/application.html.erb:10]: because I deleted the webpacker package

File: .rubocop.yml (hunk 2 above)
Comment (DmitriyFirsov) [reply .rubocop.yml]: to allow create a table with a lot of fields

Comment (DmitriyFirsov) [reply .rubocop.yml]: i removed this

**Review by KilinMikhail — APPROVED** [review]

**Pull request merged**